### ISSUE TITLE
Favourite / Bookmark World tab

### ISSUE THREAD

**Issue by HamoCorp** [Yellow-Dog-Man/Resonite-Issues#1000]
### Is your feature request related to a problem? Please describe.

Currently, the way to keep track of published worlds you like is to spawn a world orb navigate through / create folders and then save them to the inventory. Using folders to navigate through worlds doesn't have all the information on the world tab like description, active sessions and you have to remember worlds based on the name, I find it a lot easier to remember worlds based on the large 360 thumbnails you see in the world tab.

### Describe the solution you'd like

A favourite/Bookmark button on published worlds and a tab on the world page that contains favourites. also organizing favourites by creating favourite groups example: chill worlds, game worlds, worlds to visit.
I feel that doing world-related tasks like bookmarking and also things like setting home worlds would make a lot more sense to all be done from inside the world menu. This would be far more intuitive with fewer steps instead of having to go back and forth between inventory/worlds tabs and spawning out world orbs.

### Describe alternatives you've considered

Alternate ways to make navigating worlds in the inventory better: 
1. When saving world orbs to the inventory instead of pictures of the world orbs for the item thumbnail they could be saved as 2D images of the world. This would still be small and hard to see when viewed inside the inventory.

2. When selecting world orbs in the inventory there could be a button to quickly open the world menu, this would allow you to see the full thumbnail and active sessions without having to manually type in the name of the world into search each time.

3. An alternate way of doing favourites could be a favourite worlds tab where world orbs you have saved in your inventory are automatically shown on a favourites tab and groups are created based on the folder they are in. This would still require saving world orbs to the inventory and still be convoluted although better than it currently is.

### Additional Context

Other possible features could be suggested worlds based on your favourites with similar names or tags like how some browsers have suggestions based on bookmarks or how YouTube has suggested videos when creating playlists. This would make rarely visited worlds far more discoverable.

**Comment by ArchieTheRaccoon** [comment]
I agree with this requested feature! I mean it's immersive to spawn an orb and have like a place at home world for fav worlds, but very much takes too much effort and time doing this whole thing. So yeah, just a way to favourite a world and have a separate tab for it would be awesome. (Note: NOT for it to be in the inventory tab. But directly in the worlds tab!)
